src/imitation/scripts/expert_demos.py

```diff
@@ -142,6 +172,14 @@ def callback(locals_: dict, _) -> bool:
         if policy_save_interval > 0 and step % policy_save_interval == 0:
           output_dir = os.path.join(policy_dir, f'{step:05d}')
           serialize.save_stable_model(output_dir, policy, vec_normalize)
+        if ray_tune_interval > 0 and step % ray_tune_interval == 0:
+          # TODO(shwang): Not sure if `venv` has the ground truth reward
```

Comment:
`VecNormalize` clips rewards, it can also rescale them but doesn't do so by default IIRC. It's probably best to compute this before VecNormalize. 

I'm not sure on the best way to do this: we could look at the `ep_rew` added by `Monitor`? This might be too constrained, in which case we could add our own wrapper for instrumentation.

Reply:
Decided not to use `Monitor`. As discussed in [design docs](https://www.notion.so/chaiberkeley/PR-88-Transfer-learning-from-reward-models-Ray-a2dbae9c22014c8ebc14e6c1740dcc35#16acf51dd42145f9be0024973e6c1778) I decided to use Tensorboard+SB.